AdminNotificationManagerPlugin.inc.php

```diff
@@ -207,15 +226,16 @@ function _getNotificationSettingsMap() {
 	 * @return array
 	 */
 	function _getContexts() {
-		$contextDao = Application::getContextDAO();
-		$contextIterator = $contextDao->getAvailable();
-		if ($contextIterator && $contextIterator->getCount() > 1) {
-			$contextsById = array();
-			while ($context = $contextIterator->next()) {
-				$contextsById[$context->getId()] = $context->getLocalizedName();
+		$contextIdsObject = new ContextService();
+		$Ids = array();
+		if ($contextIdsObject && $contextIdsObject->getCount() >= 1) {
+			$contextsById = $contextIdsObject->getIds();
+			foreach($contextsById as $context){
+				$arrayValue = $context;
+				$Ids[$arrayValue] = $context;
```

Comment:
Similar to the above, `$arrayValue` only lives on the two lines; do we need it at all?

Reply:
In this case I feel this variable will be useful to readability when the name is fixed. Due to the more ambiguous name in this case, it is currently providing less value. With the name fixed to something more descriptive, I feel like it will be more useful